packages/component-base/src/keyboard-mixin.js

```diff
@@ -49,5 +59,27 @@ export const KeyboardMixin = dedupingMixin(
       _onKeyUp(_event) {
         // To be implemented.
       }
+
+      /**
+       * A handler for the "Enter" key. By default, it does nothing.
+       * Override the method to implement your own behavior.
+       *
+       * @param {KeyboardEvent} _event
+       * @protected
+       */
+      _onEnter(_event) {
```

Comment:
**nit:** I would give the method a more specific name like `_onEnterKeyDown` in order to avoid a possible naming conflict in the future.